apps/builder/src/features/editor/components/BlocksSideBar.tsx

```diff
@@ -49,7 +49,11 @@ export const BlocksSideBar = () => {
     x: 0,
     y: 0,
   });
-  const [isLocked, setIsLocked] = useState(true);
+  const [isLocked, setIsLocked] = useState(
+    localStorage.getItem(LOCAL_STORAGE_KEY.LEFT_SIDEBAR_LOCKED)
+      ? localStorage.getItem(LOCAL_STORAGE_KEY.LEFT_SIDEBAR_LOCKED) === "true"
+      : false,
+  );
```

Comment:
This can be simplified to

```ts
const [isLocked, setIsLocked] = useState(localStorage.getItem(LOCAL_STORAGE_KEY.LEFT_SIDEBAR_LOCKED) === "true");
```